### PR TITLE
Use © instead of (c) as the copyright sign in wxGetLibraryVersionInfo()

### PR DESCRIPTION
**Pull request by lanurmi**

### CODE REVIEW

File: src/common/utilscmn.cpp [hunk 1]
```diff
@@ -1431,12 +1431,15 @@ wxVersionInfo wxGetLibraryVersionInfo()
...
+    const wxString copyrightSign = wxString::FromUTF8("\xc2\xa9");
+
     return wxVersionInfo(wxS("wxWidgets"),
                          wxMAJOR_VERSION,
                          wxMINOR_VERSION,
                          wxRELEASE_NUMBER,
                          msg,
-                         wxS("Copyright (c) 1992-2023 wxWidgets team"));
+                         wxString::Format(wxS("Copyright %s 1992-2023 wxWidgets team"),
+                                          copyrightSign));
```
Comment (vadz) [inline src/common/utilscmn.cpp:1442]: Sorry, I don't understand how is this better than just
```suggestion
                         wxString::FromUTF8("Copyright \xc2\xa9 1992-2023 wxWidgets team"));
```
?

Comment (lanurmi) [reply src/common/utilscmn.cpp:1442]: It is only better in the sense that the variable name documents what the hex sequence stands for, but admittedly that is quite easy to guess in this context.

Comment (vadz) [reply src/common/utilscmn.cpp:1442]: OK, fair enough, it is indeed more clear, and we don't care about the extra overhead of `Format()` here nor, probably, about a few extra bytes it adds to the library... Will apply as is, thanks.

**Comment by vadz** [comment]
Applied to master now, thanks.

**Pull request closed without merging**